src/pages/index.js

```diff
@@ -13,21 +15,116 @@ import {
 
+//new api instance for getting user info
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "ea954eb7-9cc5-4b86-a221-9efd3aedab32",
+    "Content-Type": "application/json",
+  },
+});
+
+// API Fetch user information when the page loads (Step 1)
+api
+  .getUserInfo()
+  .then((userData) => {
+    // Assuming your UserInfo class only handles name and job for now
+    userInfo.setUserInfo({
+      name: userData.name,
+      job: userData.about,
+    });
+    userInfo.setAvatar(userData.avatar);
+  })
+  .catch((err) => {
+    console.error("Failed to fetch user info:", err);
+    // Optionally, show user feedback here
+  });
+
+// Fetch cards from the server (Step 2)
+let cardSection;
+
+api
+  .getInitialCards()
+  .then((cardsData) => {
+    cardSection = new Section(
+      {
+        items: cardsData,
+        renderer: (item) => {
+          const cardElement = createCard(item);
+          cardSection.addItem(cardElement);
+        },
+      },
+      ".cards__list"
+    );
+    cardSection.renderItems();
+
+    // Check if initial cards exist on the server
+    const existingCardNames = cardsData.map((card) => card.name);
+    const cardsToAdd = initialCards.filter(
+      (card) => !existingCardNames.includes(card.name)
+    );
+
+    // Only add new initial cards if they're not on the server
+    if (cardsToAdd.length > 0) {
+      return Promise.all(cardsToAdd.map((card) => api.addCard(card))).then(
+        () => {
+          console.log("New initial cards added to server.");
+          // Refresh cards after adding new ones
+          return api.getInitialCards();
+        }
+      );
+    } else {
+      console.log("All initial cards already exist on the server.");
+      return Promise.resolve(cardsData); // No need to update since no new cards were added
+    }
+  })
+  .then((updatedCardsData) => {
+    // If new cards were added, this will re-render with all cards
+    cardSection.clear();
+    cardSection.renderItems(updatedCardsData);
+  })
+  .catch((err) => {
+    console.error("Error in card operations:", err);
+  });
 // Create FormValidator instances
 const profileEditFormValidator = new FormValidator(
   validationSettings,
   profileEditForm
 );
 profileEditFormValidator.enableValidation();
 
+//form validator for add card form
 const addCardFormValidator = new FormValidator(validationSettings, addCardForm);
 addCardFormValidator.enableValidation();
 
+//form validator for avatar edit form
+const avatarEditFormValidator = new FormValidator(
+  validationSettings,
+  avatarEditForm
+);
+avatarEditFormValidator.enableValidation();
+
 /* Functions */
 
+// Create a popup for delete confirmation
+const deleteCardPopup = new PopupWithConfirmation(
+  "#delete-card-modal",
+  (cardInstance) => {
+    return api.deleteCard(cardInstance._id).then(() => {
+      cardInstance._deleteCard(); // Remove the card from the DOM after server confirmation
```

Comment:
`_deleteCard`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.